devel/code/command.py

```diff
@@ -0,0 +1,231 @@
+import argparse
```

Comment:
More than `dts devel code`, I would add this as a subcommand of the existing `dts vscode` as `dts vscode open`.

Reply:
That's an option, I intending this exclusively for a certain type of development workflow for now but we can expand it.